wandb/sdk/internal/internal_api.py

```diff
@@ -2216,6 +2216,92 @@ def upsert_run(
             server_messages,
         )
 
+    @normalize_exceptions
+    def rewind_run(
+        self,
+        run_name: str,
+        metric_name: str,
+        metric_value: float,
+        program_path: Optional[str] = None,
+        entity: Optional[str] = None,
+        project: Optional[str] = None,
+        num_retries: Optional[int] = None,
+    ) -> dict:
+        """Rewinds a run to a previous state.
+
+        Arguments:
+            run_name (str): The name of the run to rewind
+            metric_name (str): The name of the metric to rewind to
+            metric_value (float): The value of the metric to rewind to
+            program_path (str, optional): Path to the program.
+            entity (str, optional): The entity to scope this project to.
+            project (str, optional): The name of the project
+            num_retries (int, optional): Number of retries
```

Comment:
super nit: some of these don't end with a period, whereas others do. They should all consistently end with a period.

I'm also not sure whether argument types are necessary in this docstring, since they're already present in the function signature. This is an older style that existed before type annotations I believe. @kptkin do you know if this affects the generated docs?

Reply:
good question. not 100% but i think we need that with this format. cc: @ngrayluna to confirm

Reply:
This is the same format as for `upsert_run`